client-mobile/src/routing/paths.ts

```diff
@@ -1,5 +1,6 @@
 export enum Path {
   ShoppingLists = "/",
+  ShoppingList = "/shopping-lists/[id]",
```

Comment:
For the sake of clarity
```suggestion
  ShoppingListItem = "/shopping-lists/[id]",
```